tools/rpm_buildenv.sh

```diff
@@ -26,6 +26,7 @@ case "$1" in
         # builddep mixxx`).
         sudo dnf install -y \
             appstream \
+            boost-devel \
```

Comment:
What is our boost dependency on Fedora? I am not aware that boost is required on Debian based distros.

Reply:
Turns out it is actually needed on Debian-based system too, as patched in my [Devcontainer PR](https://github.com/mixxxdj/mixxx/pull/14018/files#diff-0d882499d865cf0356240933b767b1b26ca308530f71572059f0cfd18d574c25)
Boost is a dependency for `qm-dsp`, `rigtorp` and `libdjinterop`

Reply:
I never had to install it in the past, so I suspect Ubuntu-based OSes ship it by default?

Reply:
They don't depend on boost. Only the tests of libdjinterop do. But we don't deal with the tests. We use the prebuild binaries.

Reply:
Tried with a fresh box on `main` and can't reproduce so this might not be needed anymore